packages/mui-material/src/StepLabel/StepLabel.d.ts

```diff
@@ -75,6 +99,8 @@ export type StepLabelClasskey = keyof NonNullable<StepLabelProps['classes']>;
  *
  * - [StepLabel API](https://mui.com/material-ui/api/step-label/)
  */
-declare const StepLabel: ((props: StepLabelProps) => JSX.Element) & { muiName: string };
+declare const StepLabel: ((props: StepLabelProps & StepLabelSlotsAndSlotProps) => JSX.Element) & {
```

Comment:
We should do:

```
export interface StepLabelProps extends 
  StandardProps<React.HTMLAttributes<HTMLDivElement>>,
  StepLabelSlotsAndSlotProps
 {
```

instead to avoid https://github.com/mui/material-ui/pull/41324